.buildkite/pipeline.yml

```diff
@@ -0,0 +1,14 @@
+agents:
+  queue: "k8s-builders"
+
+steps:
+  - label: test tls cluster standup
+    key: aws-up
+    command: .buildkite/scripts/runtask.sh
```

Comment:
This would honestly be the saner way to do things and if I have a moment tonight I'll get it done. Task calling a script to call task is sorta dumb, but it was my response to fears about builds being canceled -- thereby stranding assets -- and needing a way to guarantee that that doesn't happen.